### ISSUE TITLE
optimumx breaks ability to update image later

### ISSUE THREAD

**Issue by andy-gutsche** [aFarkas/lazysizes#536]
Due to the fact, that optimumx removes srcset candidates from the original data-scrset, it is not possible to update images on resize or for example inside of a lightbox, because the needed resolutions had been removed already...

**Comment by aFarkas** [comment]
Yes, this is by design. but you are right improve this. Currently you would have to generate a new image. Can you please show me what you want to do exactly. The best thing would be some code and then I see what I can make possible.

But I will always have to remove the candidates otherwise the browser will load them ;-).

**Comment by aFarkas** [comment]
ach du bist es ;-)
